src/styles/components/configuration-map/styles.less

```diff
@@ -28,6 +28,7 @@
     }
 
     &-row {
+      align-items: enter;
```

Comment:
This should be `center`; I mistakenly wrote `enter` earlier today, and then joked that it wasn't a mistake... Sorry for the confusion!